.github/workflows/roundtrip/docker-compose.yaml

```
@@ -1,7 +1,7 @@
 services:
   keycloak:
     # This is kc 24.0.1 with opentdf protocol mapper on board
-    image: cgr.dev/chainguard/keycloak@sha256:37895558d2e0e93ffff75da5900f9ae7e79ec6d1c390b18b2ecea6cee45ec26f
+    image: us-docker.pkg.dev/prj-infra-automation-ktbz/remote-proxy-dockerhub/keycloak/keycloak:24.0.5
```

```suggestion
    image: keycloak/keycloak:25.0
```
dont want to reference our private proxy in opentdf

---

Thank you for the link to the right image :)